dart/test/keyword_test_keyword_test_generated.dart

```diff
@@ -0,0 +1,280 @@
+// automatically generated by the FlatBuffers compiler, do not modify
+// ignore_for_file: unused_import, unused_field, unused_element, unused_local_variable
+
+library keyword_test;
+
+import 'dart:typed_data' show Uint8List;
+import 'package:flat_buffers/flat_buffers.dart' as fb;
+
+
+class ABC {
```

Comment:
This isn't _super_ consistent in the ecosystem or SDK, but I think typically abbreviations only get the first letter of the abbreviation capitalized, with a couple quirky exceptions like `IOS`.

Reply:
I'm going to resolve this one in favor of my other comments about not changing the style too drastically in this PR. It'll be easy to change afterwards since it'll be a config change.